src/metabase/pulse/render/body.clj

```diff
@@ -763,9 +763,9 @@
         rows           (map (juxt x-axis-rowfn y-axis-rowfn)
                             (common/row-preprocess x-axis-rowfn y-axis-rowfn rows))
         labels         (x-and-y-axis-label-info x-col y-col viz-settings)
-        render-fn      (if (isa? (-> cols x-axis-rowfn :effective_type) :type/Temporal)
-                         js-svg/timelineseries-waterfall
-                         js-svg/categorical-waterfall)
+        waterfall-type (if (isa? (-> cols x-axis-rowfn :effective_type) :type/Temporal)
+                         :timeseries
+                         :categorical)
```

Comment:
I changed it to use Clojure keywords instead of strings.
It might be good to use schema to spec that this can only be one of these 2 keys, but I can add that in a PR elsewhere, and it's not a huge concern at the moment because the type here is handled internally only.

Reply:
I was using keywords before but seems it got sent as a `":timeseries"` string like this (with the leading comma). So I need to use `(name waterfall-type)` somewhere. But I trust your call on that. I don't know which one is more idiomatic.